events/code_review_events/workflow.py

```diff
@@ -134,6 +153,23 @@ def get_repositories(self, repositories, cache_root):
                 # Requeue when nothing changed for now
                 await self.bus.send(QUEUE_WEB_BUILDS, build)
 
+    def is_blacklisted(self, revision: dict):
+        """Check if the revision author is in blacklisted"""
+        author_phid = revision["fields"]["authorPHID"]
+        author = next(
```

Comment:
It'd be nicer to make self.user_blacklist a set of PHIDs, so you can simplify this

Reply:
Yes but i'll lose the naming